fastlane/Fastfile

```diff
@@ -28,9 +28,18 @@ platform :ios do
 
   desc "Increment build number"
   lane :bump do |options|
-    version_number = options[:version]
-    increment_version_number(version_number: version_number)
-    version_bump_podspec(version_number: version_number)
+    new_version_number = options[:version]
+    previous_version_number = get_version_number
+    increment_version_number(version_number: new_version_number)
+    version_bump_podspec(version_number: new_version_number)
+    increment_build_number_in_rc_purchases(previous_version_number, new_version_number)
   end
 
-end
\ No newline at end of file
+end
+
+def increment_build_number_in_rc_purchases(previous_version_number, new_version_number)
+  rc_purchases_path = '../Purchases/Public/RCPurchases.m'
+  sed_regex = 's|' + previous_version_number + '|' + new_version_number + '|'
+  backup_extension = '.bck'
+  sh("sed", '-i', backup_extension, sed_regex, rc_purchases_path)
```

Comment:
we get the old number from fastlane's `get_version_number`, the new version from the caller, and do a sed replacement on the file. It will only replace the first instance it finds.